docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -84,6 +84,42 @@ spec:
 ----
 <1> `<ciliumClusterID>` is replaced with the string provided in this parameter
 
+== `allowFromNodeLabels`
+
+[horizontal]
+type:: dict
+default:: `{}`
+
+This parameter allows users to customize the `allow-from-cluster-nodes` `CiliumNetworkPolicy`.
+
+When this parameter is empty, this policy allow access from workloads running on all cluster nodes (including nodes of other clusters in the same cluster mesh) to workloads.
+This ensures that -- for example -- an ingress controller which is running in host-network mode can access workloads that are exposed through an `Ingress`.
+
+When the parameter isn't empty, the component uses the contents as the value for `matchLabels` in a `fromNodes` entry in the `CiliumNetworkPolicy`.
+This restricts access to workloads to the host network on nodes matching the provided label selector(s).
+See the https://docs.cilium.io/en/latest/security/policy/language/#node-based[Cilium documentation] for details on the `fromNodes` policy mechanism.
+
+IMPORTANT: When setting a value for this parameter, you must ensure that you're using Cilium 1.16 or newer and that the Helm value `nodeSelectorLabels=true` is set for your Cilium installation.
```

Comment:
I assume we set  that helm value by default for our cilium installations?

Reply:
By default we're leaving this parameter empty, and we'll probably enable `nodeSelectorLabels` by default (the only downside is that traffic from remote nodes in `hubble observe` is less obvious, since that traffic now has individual identities which aren't resolved back to `remote-node` or a human-readable node name in Cilium 1.16)